packages/sdk/src/NomadContext.ts

```diff
@@ -1,13 +1,20 @@
-import { providers, Signer } from 'ethers';
+import { providers, Signer, ContractTransaction } from 'ethers';
 
 import { MultiProvider } from '@nomad-xyz/multi-provider';
 import * as core from '@nomad-xyz/contracts-core';
 import * as config from '@nomad-xyz/configuration';
 
 import { CoreContracts } from './CoreContracts';
+import { NomadMessage } from './messages/NomadMessage'
 
 export type Address = string;
 
+const s3URLs: { [key: string]: string } = {
```

Comment:
could just do 
`https://nomadxyz-${environment}-proofs.s3.us-west-2.amazonaws.com/`
but would be more brittle if the URLs did change in the future

Reply:
Well, if the urls changed, we'd have to update them anyway.  This seems like a good clean way to do it